File: 01 Introdução/1037 Intervalo/1037.py
value = float(input())

if 25 >= value >= 0:
    print("Intervalo [0,25]")
elif 25 < value <= 50:
    print("Intervalo (25,50]")
elif 50 <= value <= 75:
    print("Intervalo (50,75]")
elif 75 <= value <= 100:
    print('Intervalo (75,100]')
else:
    print('Fora de intervalo')

File: 01 Introdução/1037 Intervalo/1037.java
import java.util.Scanner;

class Main {

  public static void main(String args[]) {
    Scanner sc = new Scanner(System.in);

    double interval = sc.nextDouble();

    if (25 >= interval && interval >= 0) {
      System.out.println("Intervalo [0,25]");
    } else if (50 >= interval && interval > 25) {
      System.out.println("Intervalo (25,50]");
    } else if (75 >= interval && interval > 50) {
      System.out.println("Intervalo (50,75]");
    } else if (100 >= interval && interval > 75) {
      System.out.println("Intervalo (75,100]");
    } else {
      System.out.println("Fora de intervalo");
    }

    sc.close();

  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/1037
// Ranking 09856º, Time 0.137, Submiss Date 12/02/2023 22:58:45 (Java 14)